packages/connect-plugin-stellar/package.json

```diff
@@ -24,13 +24,13 @@
     ],
     "peerDependencies": {
         "@trezor/connect": "9.x.x",
-        "stellar-sdk": "^10.4.1"
+        "stellar-sdk": "^11.0.0-beta.3"
     },
     "devDependencies": {
         "jest": "^26.6.3",
         "rimraf": "^5.0.1",
-        "stellar-base": "^8.2.2",
-        "stellar-sdk": "^10.4.1",
+        "stellar-base": "^10.0.0-beta.1",
```

Comment:
I just released a beta.2 if you want to use that here; it shouldn’t make a material difference, though